modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js

```diff
@@ -274,7 +164,7 @@ function runOperation (operationName) {
             $(errorMsgWrapper).removeClass("hidden");
         } else {
             invokerUtil.post(serviceEndPoint, payload,
-                successCallback, function (jqXHR, textStatus, errorThrown) {
+                             successCallback, function (jqXHR, textStatus, errorThrown) {
                     console.log(textStatus);
```

Comment:
This error callback is only useful for debugging purposes, no response seems to be going to the user at the event of an error